www/include/Administration/myAccount/formMyAccount.php

```diff
@@ -81,6 +81,8 @@
     while ($row = $res->fetch()) {
         $cct[$row['cp_key']] = $row['cp_value'];
     }
+
+    $cct['default_page'] = $cct['default_page'] ?: 104; // selected by default is Events view page
```

Comment:
Please use a constant. It will be more readable

Reply:
done